sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -64,6 +64,16 @@ class AesTest : public testing::Test, public mock_mmio::MmioTest {
          {AES_CTRL_SHADOWED_SIDELOAD_BIT, options.key_sideloaded},
          {AES_CTRL_SHADOWED_FORCE_ZERO_MASKS_BIT, options.force_zero_masks}});
   }
+
+  void SetExpectedAuxConfig(bool reseed_on_key_change, bool lock) {
```

Comment:
nit: an `AuxConfigOptions` struct, similar to `ConfigOptions`, would be clearer.